docs/src/customization/theming.md

```
@@ -4,7 +4,105 @@ description: Customizing the look and feel of Citizen
+Citizen is able to customise text colors with ease.
+
+| Property             | Description                        |
+| -------------------- | ---------------------------------- |
+| `--color-base`       | Base text color (e.g. body text)   |
+| `--color-emphasized` | Emphasized text color (e.g. title) |
+| `--color-subtle`     | Subtle text color (e.g. caption)   |
+
+## Appearance
+
+Citizen provides multiple appearance options including light, dark, and pure black modes.
+| Theme            | Class    | Notes |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)